extensions/wikia/WikiaApi/WikiaApiQuerySiteinfo.php

```diff
@@ -66,7 +66,15 @@ protected function appendVariables($property) {
 		$data = array ();
 		foreach ($this->variablesList as $id => $variableName) {
 			$data[$id] = array( 'id' => $variableName );
-			$value = (array_key_exists($variableName, $GLOBALS) && !is_null($GLOBALS[$variableName])) ? $GLOBALS[$variableName] : "";
+			switch ($variableName) {
+				case 'wgWikiaGlobalUserGroups':
+					/** @var \Wikia\Service\User\Permissions\PermissionsServiceImpl $permissionsService */
+					$permissionsService = \Wikia\DependencyInjection\Injector::getInjector()->get( \Wikia\Service\User\Permissions\PermissionsService::class );
+					$value = $permissionsService->getConfiguration()->getGlobalGroups();
+					break;
+				default:
+					$value = (array_key_exists($variableName, $GLOBALS) && !is_null($GLOBALS[$variableName])) ? $GLOBALS[$variableName] : "";
+			}
 			if ( is_array($value) ) {
 				$loop = 0; foreach ($value as $key => $v) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS122)